operators/src/source/gdal_source.rs

```diff
@@ -694,29 +700,28 @@ where
         info: GdalLoadingInfoPart,
     ) -> impl Stream<Item = Result<RasterTile2D<T>>> {
         let spatial_resolution = query.spatial_resolution;
-        let geo_transform = info.params.geo_transform;
-
-        // adjust the spatial resolution to the sign of the geotransform
-        let x_signed = if geo_transform.x_pixel_size.is_sign_positive()
-            && spatial_resolution.x.is_sign_positive()
-        {
-            spatial_resolution.x
+        let no_data_value = self.no_data_value;
+
+        // TODO: re-evaluate the logic behind this
```

Comment:
maybe add a card for this. Also: what has this to do with gdal? the resolution is part of the query, not of the gdal loading info

Reply:
282ed20